Include/cpython/pyerrors.h

```diff
@@ -98,7 +98,6 @@ PyAPI_FUNC(void) _PyErr_GetExcInfo(PyThreadState *, PyObject **, PyObject **, Py
 
 /* Context manipulation (PEP 3134) */
 
-PyAPI_FUNC(void) _PyErr_ChainExceptions(PyObject *, PyObject *, PyObject *);
```

Comment:
This is marked as an API function, so it will need to be added to the "porting to 3.12" section and any other relevant sections in the "what's new".